packages/odyssey-design-tokens/src/typography.json

```diff
@@ -37,11 +37,23 @@
       }
     },
     "family": {
+      "system": {
+        "value": "'-apple-system', 'BlinkMacSystemFont'"
+      },
+      "fallBack": {
+        "value": "'Segoe UI', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif"
```

Comment:
I don't have this font `Segoe UI` installed on my Mac - where will it come from?

Reply:
cc: @lesterchoi-okta Any concerns with these fonts? I believe you've been doing some work on this to make sure the right fonts are available in the browser

Reply:
Seems like Oxygen-Sans, Ubuntu and Cantarell can all go in that system fonts definition actually. I'll make that change